docs/book/features/volume_health.md

```diff
@@ -0,0 +1,112 @@
+# vSphere CSI Driver - Volume Health
+Higher level advanced service such as Persistent Service operator wants to know the health status of persistent volumes so that it can make intelligent decisions.  Volume Health feature provides the health status of persistent volume to the higher level advanced service. Currently, Volume Health is only supported for Project Pacific Cluster and Tanzu Kubernetes Grid Cluster and only supported for block volume.
+
+## Health Status of Persistent Volume
+Currently, Volume Health only report the health status for Persistent Volumes which have been provisioned successfully. For each Persistent Volume in `bound` state, the health status of this Persistent Volume will be exposed as an annotation of Persistent Volume Claim which is bounded to the Persistent Volume. The key of the annotation is `volumehealth.storage.kubernetes.io/messages:`. There are two possible values for the health status.
+
+1. `accessible`:  The Persistent Volume is accessible and avaialbe for use.
+2. `inaccessible`: The Persistent Volume is inaccessible, user should avoid use this Persistent Volume.
+
+## RBAC Rules
+
+The following  RBAC rules need to be added to support volume health:
```

Comment:
Pacific automates all these during SV cluster and TKG deployment. Why do we need this doc?